README.md

```diff
@@ -86,6 +86,7 @@ You can see a list of these variables in [`configuration.js`](./server/configura
 Some important environment variables are :
  - `WBO_HISTORY_DIR` : configures the directory where the boards are saved. Defaults to `./server-data/`.
  - `WBO_MAX_EMIT_COUNT` : the maximum number of messages that a client can send per unit of time. Increase this value if you want smoother drawings, at the expense of being susceptible to denial of service attacks if your server does not have enough processing power. By default, the units of this quantity are messages per 4 seconds, and the default value is `192`.
+ - `AUTH_SECRET_KEY` : If you would like to authenticate your boards using jwt, this declares the secret key.
```

Comment:
You need to document how to authenticate too. How to pass the jwt to wbo ?

Reply:
Sure. I'll add an Authentication section to the Readme